packages/quick_actions/quick_actions_ios/ios/Classes/AppShortcutControlling.swift

```diff
@@ -4,12 +4,12 @@
 
 import UIKit
 
-/// Services `UIApplicationShortcutItem`s.
-protocol ShortcutItemServicing: AnyObject {
+/// Controlls app's shortcut behavior.
+protocol AppShortcutControlling: AnyObject {
```

Comment:
Why does this need `AnyObject`?

Reply:
`AnyObject` scopes this protocol to reference types (classes & closures). It prevents any value types (enums & structs) to conform to it. 

Since this is injected dependency with no value semantics, we want to scope it to reference types. See the `NOTE` below: 

```swift
class QuickActionsPlugin {
  // NOTE: If we remove `AnyObject`, we have to use `var` here. 
  private let controller: AppShortcutControlling 

  func foo() {
    controller.shortcutItems = [] // mutate here
  }
}
```